app/controllers/pages_controller.rb

```diff
@@ -83,6 +80,7 @@ def index
   end
 
   def contact
+    @fcn_link = Page.find_by_title("Frequently called numbers")
```

Comment:
@cdoyle-temple Now that we have a slug on this model, can we update the page and query by slug?